CMakeLists.txt

```diff
@@ -126,7 +126,7 @@ if(PLKOKKOS_ENABLE_PYTHON)
 FetchContent_Declare(
     pybind11
     GIT_REPOSITORY https://github.com/pybind/pybind11.git
-    GIT_TAG        v2.9.1
+    GIT_TAG        v2.10.1
```

Comment:
Also the same here. `2.10.3` is the latest version.

Reply:
see above.